cpp/ql/test/query-tests/Security/CWE/CWE-457/semmle/Errors/UninitializedLocal.qlref

```diff
@@ -0,0 +1 @@
+Likely Bugs/Memory Management/UninitializedLocal.ql
```

Comment:
The tests for this query are located in `cpp/test/query-tests/Security/CWE/CWE-457/semmle/tests`. We should probably add the test case there.

Reply:
Fine by me.